weblab/datasets/tests/test_views.py

```diff
@@ -194,6 +194,27 @@ def test_admin_can_delete_dataset(
         assert not dataset.archive_path.exists()
 
 
+@pytest.mark.django_db
+class TestTransferDatasetView:
+
+    def test_datasets_transfer_success(self, client, logged_in_user, other_user, my_dataset_with_file, helpers):
```

Comment:
As well as this success case, we should test that transfer works for a dataset without files (`my_dataset` fixture), and for error if a non-existent user, or user with a same named dataset, are given as the target.